docs/source/concepts/glossary.rst

```diff
@@ -393,24 +456,92 @@ As a user, you don't need to interact with Daml-LF directly. But internally, it'
+=====================
+
+While every participant has their own private contract store (PCS), the `Canton protocol <#canton-protocol>`__ guarantees that the contracts which are stored in the PCS are well-authorized 
+and that any change to the store is justified, authorized and valid. The result is that every participant only possesses a small part of the *virtual global ledger*. All the local 
+stores together make up that *virtual global ledger* and they are thus synchronized. The Canton protocol guarantees that the virtual ledger provides integrity, privacy, 
+transparency and auditability. The ledger is logically global, even though physically, it runs on segregated and isolated domains that are not aware of each other.
+
+.. Language for the Mediator was taken, and edited, from the Canton key terms page
+Mediator
+========
+
+The mediator is a service provided by the `domain <#domain>`__ and used by the `Canton protocol <#canton-protocol>`__. The mediator acts as commit coordinator, collecting individual transaction verdicts issued by validating 
+participants and aggregates them into a single result. The mediator does not learn about the content of the transaction, they only learn about the involved participants.
+
+.. Language for the Sequencer was taken, and edited, from the Canton key terms page
+Sequencer
+=========
+
+The sequencer is a service provided by the `domain <#domain>`__, used by the `Canton protocol <#canton-protocol>`__. The sequencer forwards encrypted addressed messages from participants and ensures that every member receives 
+the messages in the same order. Think about registered and sealed mail delivered according to the postal datestamp.
+
+Domain Identity Manager
+=======================
+
+The Domain Identity Manager is a service provided by the `domain <#domain>`__, used by the `Canton protocol <#canton-protocol>`__. Participants join a new domain by registering with the domain identity manager. The domain 
+identity manager establishes a consistent identity state among all participants. The domain identity manager only forwards identity updates. It can not invent them.
 
-When you're developing, you'll use `Sandbox <#sandbox>`__ as your ledger.
+
+Consensus
+=========
+
+The Canton protocol does not use PBFT or any similar consensus algorithm. There is no proof of work or proof of stake involved. Instead, Canton uses a variant of a stakeholder based 
+two-phase commit protocol. As such, only stakeholders of a transaction are involved in it and need to process it, providing efficiency, privacy and horizontal scalability. Canton based 
+ledgers are resilient to malicious participants as long as there is at least a single honest participant. A domain integration itself might be using the consensus mechanism of the underlying 
+platform, but participant nodes will not be involved in that process.
+
+
+
+.. General concepts
+.. ****************
 
 .. _trust-domain:
 
-Trust domain
-============
+.. Trust domain
```

Comment:
if we don’t need that, let’s delete it completely instead of commenting it out.

Reply:
Removed